### PR TITLE
add humanize method in ensemble.formatter

### PR DESCRIPTION
**Pull request by anserwaseem**
## Describe your changes
Add `humanize` method to ensemble formatter

Use it like `ensemble.formatter.now().humanize(date)`

## Issue ticket number and link

Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added tests
- [x] I have added a changeset `pnpm changeset add`
- [x] I have added example usage in the kitchen sink app

### CODE REVIEW

**Review by justEhmadSaeed — APPROVED** [review]
lgtm

File: packages/framework/src/date/dateFormatter.ts [hunk 1]
```diff
@@ -60,6 +66,8 @@ export const DateFormatter = (): EnsembleFormatter => {
         dayjs(input).diff(currentDate, "month"),
       getYearsDifference: (input: string): number =>
         dayjs(input).diff(currentDate, "year"),
+      humanize: (input: string): string =>
```
Comment (evshi) [inline packages/framework/src/date/dateFormatter.ts:69]: I think this should be a bit more descriptive - like humanizeDifferenceFromToday? Or, the other option is to make it composable/chainable with the other smaller functions.

Comment (anserwaseem) [reply packages/framework/src/date/dateFormatter.ts:69]: We'll already be using it composed of `now()` so it's already known that difference is from the current date.
Maybe we should expose same function in `EnsembleFormatter` as well which would take source date as input too. What do you say?

Comment (evshi) [reply packages/framework/src/date/dateFormatter.ts:69]: I think this is probably fine for now. We can always add an optional second param later. Just to clarify when I say compose - I mean allow the user to compose it with other day js functions, not just internally.

**Review by evshi — APPROVED** [review]

**Pull request merged**